Makefile

```diff
@@ -19,7 +19,7 @@ BUILD_OUTPUT_DIRECTORY = $(BUILD_DIRECTORY)/out
 # ---------------------------------------------------------------------
 
 ELECTRON_VERSION = $(shell jq -r '.devDependencies["electron"]' package.json)
-NODE_VERSION = 6.10.3
+NODE_VERSION = 6.1.0
```

Comment:
What impact does switching to a lower nodeJS version have?
ISTR you saying in gitter that the version of `node-pre-gyp` we're using doesn't work with newer node versions - would upgrading to a newer version of `node-pre-gyp` fix that (and allow us to use Node 6.10.3), or does upgrading `node-pre-gyp` come with other problems?